docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -392,6 +385,16 @@ In general, to reduce complexity, reduce the number of objects requested:
 * reduce the number of items in nested collections
 * request less fields
 
+<a id="see-it-in-action" class="devdocsAnchor"></a>
+
+## See it in Action
+
+To see the GraphQL storefront API in action, checkout the [Bootstrap + Vanilla JS Storefront API Example](https://bigcommerce.github.io/storefront-api-examples/html-bootstrap-vanillajs/) hosted on GitHub. This example shows how a static HTML site can be used to render dynamic product information via the GraphQL Storefront API.
+
+Simply open the link and click submit with the sample data in the form. To see the example page with your store's data, [create a Storefront API Token](https://developer.bigcommerce.com/api-reference/storefront/storefront-token-api/api-token/createtoken) against your store and paste the token into the example form (be sure to create a token valid for this origin: `https://bigcommerce.github.io`).

```

Comment:
I suggest removing "simply" here. It's not necessary and could be construed as condescending.